context/src/test/groovy/io/micronaut/scheduling/ScheduledFixedRateSpec.groovy

```diff
@@ -131,13 +134,15 @@ class ScheduledFixedRateSpec extends Specification {
     }
 
     @Singleton
+    @Requires(property = 'spec.name', value = 'ScheduledFixedRateSpecMyOtherTask')
```

Comment:
Without this annotation, this Singleton is active for other tests that use `scheduled-test.task.enabled` too!